.github/workflows/universalnrp-test.yml

```diff
@@ -2,13 +2,48 @@ name: Universal NRP Test
 
 on:
   workflow_dispatch:
+    inputs:
+      policyPackageList:
+        description: 'List of policy packages to test'
+        required: true
+        default: '[]'
```

Comment:
I think this can be greatly simplified by adding the defaults here and keeping the rest of the matrix instead of having JSON and merging the input/default value.

Reply:
i originally did that, problem is the workflow inputs are not defined on PRs so that would force me to define it twice. I went with a single definition through the environment variables